src/App.tsx

```diff
@@ -1,24 +1,183 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { addTodos, deleteTodos, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorTypes } from './types/ErrorTypes';
+import { ErrorMessage } from './components/ErrorMessage';
+import { TodoItem } from './components/TodoItem';
 
-const USER_ID = 0;
+const USER_ID = 10905;
 
 export const App: React.FC = () => {
+  const [title, setTitle] = useState<string>('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedFilter, setSelectedFilter] = useState('All');
```

Comment:
Consider using an enum for the filter types to prevent mistakes associated with string values.

Reply:
done